docs/docs/usage_instructions/retrieve_available.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 1
+---
+
+# Retrieving available items
+
+First thing you should do is to define your product IDs for iOS and Android separately like defined below.
+
+```javascript
+import * as RNIap from 'react-native-iap';
+
+const productIds = Platform.select({
+  ios: [
+    'com.example.coins100'
+  ],
+  android: [
+    'com.example.coins100'
+  ]
+});
+```
+
+To get a list of valid items, call `getProducts()`.
+
+You can do it in `componentDidMount()`, or another area as appropriate for you app.
```

Comment:
`you` => `your`. Already mistake from previous `readme` tough.

Reply:
```suggestion
You can do it in `componentDidMount()`, or another area as appropriate for your app.
```